src/main/java/org/jitsi/videobridge/rest/shutdown/Shutdown.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.rest.shutdown;
+
+import com.fasterxml.jackson.annotation.*;
+import org.jitsi.videobridge.util.*;
+import org.jitsi.xmpp.extensions.colibri.*;
+import org.jxmpp.jid.impl.*;
+import org.jxmpp.stringprep.*;
+
+import javax.servlet.http.*;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+
+@Path("/")
+public class Shutdown
+{
+    protected final VideobridgeProvider videobridgeProvider;
+    public Shutdown(VideobridgeProvider videobridgeProvider)
+    {
+        this.videobridgeProvider = videobridgeProvider;
+    }
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response shutdown(ShutdownJson shutdown, @Context HttpServletRequest request) throws XmppStringprepException
+    {
+        ShutdownIQ shutdownIq = shutdown.toIq();
+
+        String ipAddress = request.getHeader("X-FORWARDED-FOR");
+        if (ipAddress == null)
+        {
+            ipAddress = request.getRemoteAddr();
+        }
+        shutdownIq.setFrom(JidCreate.from(ipAddress));
+        videobridgeProvider.get().handleShutdownIQ(shutdownIq);
+
+        return Response.ok().build();
```

Comment:
The previous code had:
```  
              IQ responseIQ	
                    = videobridge.handleShutdownIQ(	
                            requestShutdownIQ);	

                if (IQ.Type.result.equals(responseIQ.getType()))	
                {	
                    status = HttpServletResponse.SC_OK;	
                }	
                else	
                {	
                    status = getHttpStatusCodeForResultIq(responseIQ);	
                }
        ...
        response.setStatus(status);	
```

Should we be handling IQ error responses similarly here?

Reply:
yep, i think we should...missed this and thought it just threw.  i'll change it.